tiledb/sm/query/readers/sparse_unordered_with_dups_reader.cc

```diff
@@ -173,6 +173,12 @@ Status SparseUnorderedWithDupsReader<BitmapType>::initialize_memory_budget() {
 
 template <class BitmapType>
 Status SparseUnorderedWithDupsReader<BitmapType>::dowork() {
+  // Ensure we include coordinates for multipart remote queries.
```

Comment:
Let's have the constructor set include_coords_ to false and always set it here regardless of remove array or not.